components/cam/src/physics/cam/micro_mg2_0.F90

```diff
@@ -1217,6 +1207,11 @@ subroutine micro_mg_tend ( &
            qcic(i,k)=min(qc(i,k)/lcldm(i,k),5.e-3_r8)
            ncic(i,k)=max(nc(i,k)/lcldm(i,k),0._r8)
 
+           ! impose minimum droplet number conc (in-cloud) 
+           if (mincdnc.gt.0.) then
```

Comment:
To be consistent with other lines and the variable type,  how about 
if (mincdnc.gt.0._r8) then

Reply:
Thanks for the suggestion. It's fixed.